### PR TITLE
TA-39: Added error message in text-area component

### PR DESCRIPTION
**Pull request by soyeba3**
@mbmohib vaiya,

## I have added error message in text-area component

### CODE REVIEW

File: src/app.tsx [hunk 1]
```diff
@@ -14,14 +10,7 @@ export default function App() {
       </h1>
       <div id="playground">
         {/* Play inside this div */}
```
Comment (Sayemhaque) [inline src/app.tsx:12]: Do we need this comment?

File: src/components/text-area/text-area.tsx [hunk 2]
```diff
@@ -21,20 +22,29 @@ export type TextareaProps = React.TextareaHTMLAttributes<HTMLTextAreaElement> &
...
           rows={rows}
           cols={cols}
           placeholder={placeholder}
           ref={ref}
           {...props}
         />
+        {error && <ErrorMessage message={error} />}
         <p className="font-normal text-gray-600">
           Please don’t exceed 300 characters.
         </p>
```
Comment (mbmohib) [inline src/components/text-area/text-area.tsx]: @soyeba3  what if in another place we want to have the limit of 500 chars?

File: src/app.tsx (hunk 1 above)
Comment (soyeba3) [reply src/app.tsx:12]: @Sayemhaque Yes. It's a hint for new dev

**Review by Sayemhaque — APPROVED** [review]

File: src/components/text-area/text-area.tsx (hunk 2 above)
Comment (soyeba3) [reply src/components/text-area/text-area.tsx]: @mbmohib vaiya, Please check the latest commit, we are receiving hint text as a props now

**Pull request merged**